### PR TITLE
Loa 394 implement user page styling

### PR DESCRIPTION
**Pull request by Kostanix**

### CODE REVIEW

File: app/src/main/res/layout/fragment_user_page.xml [hunk 1]
```diff
@@ -4,133 +4,237 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="65dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+
+        <LinearLayout
+            android:id="@+id/badges_page_linearlayout"
```
Comment (SebastianMaier03) [inline app/src/main/res/layout/fragment_user_page.xml:21]: why is in the user page a badge page linearlayout?

**Review by SebastianMaier03 — CHANGES_REQUESTED** [review]
pls change the issiue

File: app/src/main/res/layout/fragment_user_page.xml (hunk 1 above)
Comment (Kostanix) [reply app/src/main/res/layout/fragment_user_page.xml]: My mistake, will fix

**Review by SebastianMaier03 — APPROVED** [review]
LGTM

**Pull request merged**